### PR TITLE
chore: Added all the listed Blogs

### LINKED ISSUE #1: Write blog article for setup CI/CD in gitlab

### LINKED ISSUE #21: Write blog article about licencing

### LINKED ISSUE #22: Write blog article about open source

### LINKED ISSUE #23: Write blog article about self hosted

### LINKED ISSUE #24: Write blog article about capgo purpose

### LINKED ISSUE #39: add dpa page
like plausible:
https://plausible.io/dpa

### LINKED ISSUE #40: Add data policy

### PR DESCRIPTION
**Pull request by anikdhabal**
fixes #1 
fixes #21 
fixes #22 
fixes #23 
fixes #24 
fixes #39 
fixes #40 

Added all the listed blogs and two pages with best possible way.
Screenshot 2023-09-16 182516

### CODE REVIEW

**Comment by anikdhabal** [comment]
@riderx can you review it once?

File: src/content/blog/open-source-licence.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+* If you're unsure which license to choose, the Open Source Initiative has a list of popular licenses you can compare.
+
+Choosing the correct license for your open source project is an important decision. By carefully considering your needs and goals, you can choose a license that will help you to achieve your desired outcomes.
+
+## Capgo new license
+
+How do I accomplish all that? By changing my license. 
+
+Capgo has now changed the license from the MIT to a newer licensing scheme called GNU Affero General Public License V3 (AGPLv3) or any later version. Capgo version can be [found here](https://github.com/cap-go/capgo/blob/main/LICENSE.md).
```
Comment (riderx) [inline src/content/blog/open-source-licence.md:133]: Wrong link it should be from capacitor updater repo, and the licence type is not correct

Comment (anikdhabal) [reply src/content/blog/open-source-licence.md:133]: Ok

**Comment by riderx** [comment]
@anikdhabal thanks a lot that super nice ! 
just check the link issue and then i merge and tip you :)

**Comment by anikdhabal** [comment]
> @anikdhabal thanks a lot that super nice ! just check the link issue and then i merge and tip you :)

Okk let me fix it.

**Comment by anikdhabal** [comment]
> Can you make the date of article in different dates ?

Ok

**Comment by anikdhabal** [comment]
@riderx Done

**Comment by anikdhabal** [comment]
@riderx done

**Comment by riderx** [comment]
The article For open source are meant to be related to Capgo

**Comment by riderx** [comment]
It's missing the issue 23 I think.
I will tip you $25 by article, thanks for the work !

**Comment by anikdhabal** [comment]
> It's missing the issue 23 I think. I will tip you $25 by article, thanks for the work !

Ok I'm going to update those articles and made the final commit. Thanks for the review @riderx .

**Comment by anikdhabal** [comment]
@riderx I've added the issue 23 and also update the GitLab article.

**Comment by anikdhabal** [comment]
@riderx I've added the issue 23 and also update the GitLab article. Apart from the blogs I've also added two mentioned pages.

**Pull request merged**